openfisca_us/variables/demographic/person.py

```diff
@@ -43,3 +43,57 @@ def formula(person, period, parameters):
             [age < 18, age < 65, age >= 65],
             [AgeGroup.CHILD, AgeGroup.WORKING_AGE, AgeGroup.SENIOR],
         )
+
+
+class ccdf_age(Variable):
+    value_type = float
+    entity = Person
+    label = u"CCDF age"
+    definition_period = YEAR
+
+
+class CCDFCareLocation(Enum):
+    CENTER_BASED = "Center-based"
+    HOME_BASED = "Home-based"
+
+
+class ccdf_care_location(Variable):
+    value_type = Enum
+    possible_values = CCDFCareLocation
+    default_value = CCDFCareLocation.CENTER_BASED
+    entity = Person
+    label = u"CCDF care location"
+    definition_period = YEAR
+
+
+class CCDFAgeGroup(Enum):
+    I = "Infant"
+    T = "Toddler"
+    PS = "Preschooler"
+    SA = "School age"
+
+
+class ccdf_age_group(Variable):
+    value_type = Enum
+    possible_values = CCDFAgeGroup
+    default_value = CCDFAgeGroup.I
+    entity = Person
+    label = u"CCDF age group"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        ccdf_age = person("ccdf_age", period)
+        ccdf_care_location = person(
+            "ccdf_care_location", period
+        ).decode_to_str()
+        return select(
+            [
+                (ccdf_age < 1.5 and ccdf_care_location == "CENTER_BASED")
```

Comment:
Could you use `&` and `|` instead of `and` and `or`? I think the vectorisation requires it.